src/solvers/smt/tuple/smt_tuple_sym.cpp

```diff
@@ -115,6 +115,12 @@ smt_astt smt_tuple_sym_flattener::tuple_array_create(
   return newsym;
 }
 
+expr2tc smt_tuple_sym_flattener::tuple_get(const type2tc &, smt_astt)
+{
+  msg.error("not implemented");
```

Comment:
Any particular reason?

Reply:
I do not know how to implement this function, still digging through the smt-tuple-sym code.

Reply:
Can we just print the error message and move on?

I find it very frustrating when esbmc crashes in the last step of the verification: printing the cex.

Reply:
Sure, I'll remove the abort() and just return an empty expression as is done in the tuple-node-flattener. I could not construct an example that invoked this function, yet.

Reply:
Done.